Dockerfile

```diff
@@ -1,23 +1,23 @@
-FROM golang:1.9.1-alpine
-
-# Common ENV
-ENV API_SECRET="This is a local API secret for everyone. BsscSHqSHiwrBMJsEGqbvXiuIUPAjQXU" \
-    SERVER_SECRET="This needs to be the same secret everywhere. YaHut75NsK1f9UKUXuWqxNN0RUwHFBCy" \
-    LONGTERM_KEY="abcdefghijklmnopqrstuvwxyz" \
-    DISCOVERY_HOST=hakken:8000 \
-    PUBLISH_HOST=hakken \
-    METRICS_SERVICE="{ \"type\": \"static\", \"hosts\": [{ \"protocol\": \"http\", \"host\": \"highwater:9191\" }] }" \
-    USER_API_SERVICE="{ \"type\": \"static\", \"hosts\": [{ \"protocol\": \"http\", \"host\": \"shoreline:9107\" }] }" \
-    SEAGULL_SERVICE="{ \"type\": \"static\", \"hosts\": [{ \"protocol\": \"http\", \"host\": \"seagull:9120\" }] }" \
-    GATEKEEPER_SERVICE="{ \"type\": \"static\", \"hosts\": [{ \"protocol\": \"http\", \"host\": \"gatekeeper:9123\" }] }" \
-# Container specific ENV
-    TIDEPOOL_TIDE_WHISPERER_ENV="{\"hakken\": { \"host\": \"hakken:8000\" },\"gatekeeper\": { \"serviceSpec\": { \"type\": \"static\", \"hosts\": [\"http://gatekeeper:9123\"] } },\"seagull\": { \"serviceSpec\": { \"type\": \"static\", \"hosts\": [\"http://seagull:9120\"] } },\"shoreline\": {\"serviceSpec\": { \"type\": \"static\", \"hosts\": [\"http://shoreline:9107\"] },\"name\": \"tide-whisperer-local\",\"secret\": \"This needs to be the same secret everywhere. YaHut75NsK1f9UKUXuWqxNN0RUwHFBCy\",\"tokenRefreshInterval\": \"1h\"}}" \
-    TIDEPOOL_TIDE_WHISPERER_SERVICE="{\"service\": {\"service\": \"tide-whisperer-local\",\"protocol\": \"http\",\"host\": \"localhost:9127\",\"keyFile\": \"config/key.pem\",\"certFile\": \"config/cert.pem\"},\"mongo\": {\"connectionString\": \"mongodb://mongo/data\"},\"schemaVersion\": {\"minimum\": 1,\"maximum\": 99}}"
+# Development
+FROM golang:1.9.2-alpine AS development
 
 WORKDIR /go/src/github.com/tidepool-org/tide-whisperer
 
-COPY . /go/src/github.com/tidepool-org/tide-whisperer
+COPY . .
 
-RUN  ./build.sh && rm -rf .git .gitignore
+RUN  ./build.sh
 
 CMD ["./dist/tide-whisperer"]
```

Comment:
Not needed.

Reply:
Yes, it is. You can have Docker Compose target and use the "development" target of the Dockerfile. That way, the container in use by Docker Compose still has all of the development tools installed (as in, it isn't just the binary).

The upcoming `docker-compose.yml` file and README.md (hopefully later today depending upon meetings) will explain.

Thus, leaving as-is.

Reply:
Please approve.